CHANGES.rst

```diff
@@ -1,3 +1,73 @@
+23.2.0 (To be determined)
+=========================
+New feature release in the 23.2.x series.
+
+This release wraps up a significant refactor of fMRIPrep. The main new features
+can be used with the ``--level`` and ``--derivatives`` flags.
+
+The ``--level`` flag can take the arguments ``minimal``, ``resampling`` or
+``full``. The default is ``full``, which should produce nearly the same results
```

Comment:
I'm curious: what is the difference that makes the results nearly-but-not-exactly the same?

Reply:
```diff
diff --git a/.circleci/ds005_bids_outputs.txt b/.circleci/ds005_outputs.txt
index e7f47359..167d82ee 100644
--- a/.circleci/ds005_bids_outputs.txt
+++ b/.circleci/ds005_outputs.txt
@@ -10,33 +10,32 @@ logs/CITATION.md
 logs/CITATION.tex
 sub-01
 sub-01/anat
-sub-01/anat/sub-01_desc-aparcaseg_dseg.nii.gz
-sub-01/anat/sub-01_desc-aseg_dseg.nii.gz
 sub-01/anat/sub-01_desc-brain_mask.json
 sub-01/anat/sub-01_desc-brain_mask.nii.gz
 sub-01/anat/sub-01_desc-preproc_T1w.json
 sub-01/anat/sub-01_desc-preproc_T1w.nii.gz
+sub-01/anat/sub-01_desc-ribbon_mask.json
 sub-01/anat/sub-01_desc-ribbon_mask.nii.gz
 sub-01/anat/sub-01_dseg.nii.gz
 sub-01/anat/sub-01_from-fsnative_to-T1w_mode-image_xfm.txt
 sub-01/anat/sub-01_from-MNI152NLin2009cAsym_to-T1w_mode-image_xfm.h5
 sub-01/anat/sub-01_from-T1w_to-fsnative_mode-image_xfm.txt
 sub-01/anat/sub-01_from-T1w_to-MNI152NLin2009cAsym_mode-image_xfm.h5
-sub-01/anat/sub-01_hemi-L_curv.shape.gii
 sub-01/anat/sub-01_hemi-L_desc-reg_sphere.surf.gii
-sub-01/anat/sub-01_hemi-L_inflated.surf.gii
 sub-01/anat/sub-01_hemi-L_midthickness.surf.gii
 sub-01/anat/sub-01_hemi-L_pial.surf.gii
+sub-01/anat/sub-01_hemi-L_space-fsLR_desc-msmsulc_sphere.surf.gii
 sub-01/anat/sub-01_hemi-L_space-fsLR_desc-reg_sphere.surf.gii
+sub-01/anat/sub-01_hemi-L_sphere.surf.gii
 sub-01/anat/sub-01_hemi-L_sulc.shape.gii
 sub-01/anat/sub-01_hemi-L_thickness.shape.gii
 sub-01/anat/sub-01_hemi-L_white.surf.gii
-sub-01/anat/sub-01_hemi-R_curv.shape.gii
 sub-01/anat/sub-01_hemi-R_desc-reg_sphere.surf.gii
-sub-01/anat/sub-01_hemi-R_inflated.surf.gii
 sub-01/anat/sub-01_hemi-R_midthickness.surf.gii
 sub-01/anat/sub-01_hemi-R_pial.surf.gii
+sub-01/anat/sub-01_hemi-R_space-fsLR_desc-msmsulc_sphere.surf.gii
 sub-01/anat/sub-01_hemi-R_space-fsLR_desc-reg_sphere.surf.gii
+sub-01/anat/sub-01_hemi-R_sphere.surf.gii
 sub-01/anat/sub-01_hemi-R_sulc.shape.gii
 sub-01/anat/sub-01_hemi-R_thickness.shape.gii
 sub-01/anat/sub-01_hemi-R_white.surf.gii
@@ -44,24 +43,48 @@ sub-01/anat/sub-01_label-CSF_probseg.nii.gz
 sub-01/anat/sub-01_label-GM_probseg.nii.gz
 sub-01/anat/sub-01_label-WM_probseg.nii.gz
 sub-01/func
-sub-01/func/sub-01_task-mixedgamblestask_run-01_boldref.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-brain_mask.json
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-brain_mask.nii.gz
 sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-confounds_timeseries.json
 sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-confounds_timeseries.tsv
-sub-01/func/sub-01_task-mixedgamblestask_run-01_from-scanner_to-boldref_mode-image_xfm.txt
-sub-01/func/sub-01_task-mixedgamblestask_run-01_from-scanner_to-T1w_mode-image_xfm.txt
-sub-01/func/sub-01_task-mixedgamblestask_run-01_from-T1w_to-scanner_mode-image_xfm.txt
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-coreg_boldref.json
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-coreg_boldref.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-hmc_boldref.json
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-hmc_boldref.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-preproc_bold.json
+sub-01/func/sub-01_task-mixedgamblestask_run-01_desc-preproc_bold.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-01_from-boldref_to-T1w_mode-image_desc-coreg_xfm.json
+sub-01/func/sub-01_task-mixedgamblestask_run-01_from-boldref_to-T1w_mode-image_desc-coreg_xfm.txt
+sub-01/func/sub-01_task-mixedgamblestask_run-01_from-orig_to-boldref_mode-image_desc-hmc_xfm.json
+sub-01/func/sub-01_task-mixedgamblestask_run-01_from-orig_to-boldref_mode-image_desc-hmc_xfm.txt
 sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-L_space-fsaverage5_bold.func.gii
 sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-L_space-fsaverage5_bold.json
+sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-L_space-fsnative_bold.func.gii
+sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-L_space-fsnative_bold.json
 sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-R_space-fsaverage5_bold.func.gii
 sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-R_space-fsaverage5_bold.json
-sub-01/func/sub-01_task-mixedgamblestask_run-02_boldref.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-R_space-fsnative_bold.func.gii
+sub-01/func/sub-01_task-mixedgamblestask_run-01_hemi-R_space-fsnative_bold.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-brain_mask.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-brain_mask.nii.gz
 sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-confounds_timeseries.json
 sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-confounds_timeseries.tsv
-sub-01/func/sub-01_task-mixedgamblestask_run-02_from-scanner_to-boldref_mode-image_xfm.txt
-sub-01/func/sub-01_task-mixedgamblestask_run-02_from-scanner_to-T1w_mode-image_xfm.txt
-sub-01/func/sub-01_task-mixedgamblestask_run-02_from-T1w_to-scanner_mode-image_xfm.txt
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-coreg_boldref.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-coreg_boldref.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-hmc_boldref.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-hmc_boldref.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-preproc_bold.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_desc-preproc_bold.nii.gz
+sub-01/func/sub-01_task-mixedgamblestask_run-02_from-boldref_to-T1w_mode-image_desc-coreg_xfm.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_from-boldref_to-T1w_mode-image_desc-coreg_xfm.txt
+sub-01/func/sub-01_task-mixedgamblestask_run-02_from-orig_to-boldref_mode-image_desc-hmc_xfm.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_from-orig_to-boldref_mode-image_desc-hmc_xfm.txt
 sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-L_space-fsaverage5_bold.func.gii
 sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-L_space-fsaverage5_bold.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-L_space-fsnative_bold.func.gii
+sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-L_space-fsnative_bold.json
 sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-R_space-fsaverage5_bold.func.gii
 sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-R_space-fsaverage5_bold.json
+sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-R_space-fsnative_bold.func.gii
+sub-01/func/sub-01_task-mixedgamblestask_run-02_hemi-R_space-fsnative_bold.json
 sub-01.html
```